testing/terraform/APKBUILD

```diff
@@ -1,75 +1,44 @@
 # Contributor: Thomas Boerger <thomas@webhippie.de>
+# Contributor: Gennady Feldman <gena01@gmail.com>
 # Maintainer: Thomas Boerger <thomas@webhippie.de>
 pkgname=terraform
-pkgver=0.6.16
-pkgrel=1
+pkgver=0.7.9
+pkgrel=0
 pkgdesc="Building, changing, and combining infrastructure safely and efficiently"
 url="https://www.terraform.io/"
 arch="x86_64"
 license="MPL 2.0"
 depends=""
-depends_dev=""
-makedepends="$depends_dev go go-tools bash"
+makedepends="go tar"
 install=""
-subpackages=""
+options="!strip"
 source="${pkgname}-${pkgver}.tar.gz::https://github.com/hashicorp/terraform/archive/v$pkgver.tar.gz"
-
 builddir="${srcdir}/terraform-${pkgver}"
-_godir="${srcdir}/go"
-_gourl="github.com/hashicorp/terraform"
-
-_providers="provider-atlas provider-aws provider-azure provider-azurerm
-	provider-chef provider-clc provider-cloudflare provider-cloudstack
-	provider-cobbler provider-consul provider-datadog provider-digitalocean
-	provider-dme provider-dnsimple provider-docker provider-dyn provider-fastly
-	provider-github provider-google provider-heroku provider-influxdb
-	provider-librato provider-mailgun provider-mysql provider-null
-	provider-openstack provider-packet provider-postgresql provider-powerdns
-	provider-rundeck provider-softlayer provider-statuscake provider-template
-	provider-terraform provider-tls provider-triton provider-ultradns
-	provider-vcd provider-vsphere"
+projdir="$builddir/src/github.com/hashicorp/$pkgname"
 
-_provisioners="provisioner-chef provisioner-file provisioner-local-exec
-	provisioner-remote-exec"
+unpack() {
+	if [ -z "$force" ]; then
+		verify || return 1
+		initdcheck || return 1
+	fi
 
+	mkdir -p "$projdir"
+	msg "Unpacking sources..."
+	tar -C "$projdir" --strip-components=1 -xzf $pkgname-$pkgver.tar.gz || return 1
+}
```

Comment:
This does not make it cleaner. `unpack` function should not be normally override with custom implementation without calling `default_unpack`, it may cause many problems to future.